docs/source-2.0/aws/aws-iam.rst

```diff
@@ -139,6 +139,9 @@ Each condition key structure supports the following members:
     * - externalDocumentation
       - ``string``
       - A valid URL that defines more information about the condition key.
+    * - relativeDocumentation
+      - ``string``
+      - A relative URL that defines more information about the condition key.
```

Comment:
Could these docs (and the ones that mimic them in the `.smithy` definition and Java code) be updated slightly:

>A relative URL path that defines more information about the \<INSERT TYPE HERE> within a set of IAM-related documentation.